src/index.html

```diff
@@ -2,12 +2,85 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+    </div>
```

Comment:
to me it seems that you should have a list with list items here. `div` makes no sense, so it's better to add some semantics here

Reply:
and also, as those stars blocks are on the same level of nesting and there are quite a lot of them - you can add some empty lines between them for better readability